gender_novels/deployment/templates/gender_novels.html

```diff
@@ -7,5 +7,59 @@
 {% endblock %}
 
 {% block content %}
-    <p class="header-text">Gender Novels</p>
+    <!-- Header -->
+    <div class="container-fluid header">
+        <div class="row justify-content-center">
+            <div class="col-auto">
+                <h1 class="header-text display-2">The Gender Novels Project</h1>
+            </div>
+            <div class="col-auto align-center align-self-center">
+                <a href="https://github.com/dhmit/gender_novels" target="_blank">
+                    <img src="{{ url_for('static', filename='graphics/GitHub_light.png') }}"
+                         class="github-logo">
+                </a>
+            </div>
+        </div>
+    </div>
+
+    <!-- Topic Content -->
+    <!-- Topic Title -->
+    <div class="container-fluid">
+        <div class="row justify-content-center align-self-center">
+            <div class="col-auto">
+                <h2 id="topic-title" class="display-4">Overview</h2>
+            </div>
+        </div>
+        <!-- Topics List -->
+        <div class="row justify-content-start">
+            <div class="col-md-3">
+                <ul class="list-group list-group-flush sidebar">
+                    <a href="#topic-title" class="list-group-item
+                    list-group-item-action"><span>Overview</span></a>
+                    <a href="#topic-title" class="list-group-item
+                    list-group-item-action"><span>Topic 2</span></a>
+                </ul>
+            </div>
+            <!-- Topic Content -->
+            <div class="col" align="center">
+                <p class="lead">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris laoreet neque ac
+                    quam gravida, convallis pellentesque urna pharetra. Nullam viverra felis eu odio
+                    iaculis, nec imperdiet nisi sollicitudin. Aenean ut velit quis nisl molestie
+                    condimentum quis et est. Etiam non imperdiet nunc. Morbi non commodo elit.
+                    Praesent ultricies sollicitudin est, et dictum ante ultricies non.
+                    Ut fringilla erat nunc, et luctus dolor bibendum sed.
+                    Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac
+                    turpis egestas. Sed ante odio, porta a ligula nec, molestie aliquam sapien.
+                    Sed vel suscipit nisl, pharetra mattis augue. Phasellus sed malesuada purus,
+                    non bibendum augue. Sed posuere neque a augue laoreet, non molestie nunc volutpat.
+                    Quisque pharetra enim eu porta tristique. Proin dictum metus ante.
+                    Vivamus volutpat ac erat ut tempus. Pellentesque id semper sapien.
+                </p>
+            </div>
+            <div class="col-md-3">
```

Comment:
The sidebar already takes up 3 cols in the gridsystem so you don't need this div.